src/django_mysql/models/aggregates.py

```diff
@@ -28,6 +28,7 @@ class GroupConcat(Aggregate):
     def __init__(
         self,
         expression: Expression,
+        filter=None,
```

Comment:
We require type hints. We can copy from django-stubs here ( https://github.com/typeddjango/django-stubs/blob/master/django-stubs/db/models/aggregates.pyi ) and just use `Any | None`, which isn't much of a type hint, but if django-stubs gets better later, we can copy it.

```suggestion
        filter: Any | None=None,
```